### ISSUE TITLE
Wyoming can clip some long values

### ISSUE THREAD

**Issue by dopplershift** [Unidata/siphon#315]
For the following data, the `height` column ends up incorrectly reading the height values:

```python
from datetime import datetime
from siphon.simplewebservice.igra2 import WyomingUpperAir

df = WyomingUpperAir.request_data(datetime(2021, 1, 5, 12), '03005')
```

This is because by default `read_fwf` only looks at the first 100 rows, and for this dataset heights >= 10000m are beyond that number of rows.

**Comment by dopplershift** [comment]
The easy fix for this, based on the [pandas docs](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_fwf.html) is to pass `infer_rows=200`.

**Comment by jrleeman** [comment]
Can't reproduce this issue - formatting appears correct for the example given, though the import should read

```
from siphon.simplewebservice.wyoming import WyomingUpperAir
```

**Comment by dopplershift** [comment]
Fixed by #696.
